tests/functional/local-overlay-store/common.sh

```diff
@@ -69,8 +71,10 @@ mountOverlayfs () {
     || skipTest "overlayfs is not supported"
 
   cleanupOverlay () {
+    # shellcheck disable=2317
     umount -n "$storeBRoot/nix/store"
-    rm -r $storeVolume/workdir
+    # shellcheck disable=2317
+    rm -r "$storeVolume"/workdir
```

Comment:
@Ericson2314 this is the only one that concerned me that it might be a "legitimate bug" that was found.